deploy/fat_container/templates/nginx_app.conf.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -o nounset
+
+# In the config file, there's three types of variables, all represented with the syntax `$name`. The ones that are not
+# escaped with a backslash are rendered within this script. Among the ones that are escaped with a backslash, the ones
+# starting with `APPSMITH_` will be rendered at boot-up time by appsmith-editor docker container. The rest (like $scheme
+# and $host) are for nginx to work out.
+
+NGINX_SSL_CMNT="$1"
+CUSTOM_DOMAIN="$2"
+
+cat <<EOF
+server {
+    listen 80;
+$NGINX_SSL_CMNT    server_name $CUSTOM_DOMAIN ;
+    client_max_body_size 100m;
+
+    gzip on;
+
+    root /opt/appsmith/editor;
+    index index.html index.htm;
+
+    location /.well-known/acme-challenge/ {
+        # root /var/www/certbot;
+		root /opt/appsmith/data/certificate/certbot;
```

Comment:
If the commented line is not required, or is unused, can we remove it? Let's try to avoid commented code where possible.

```suggestion
		root /opt/appsmith/data/certificate/certbot;
```